packages/tools/fluid-runner/src/index.ts

```diff
@@ -5,3 +5,4 @@
 
 export * from "./codeLoaderBundle";
 export * from "./exportFile";
+export * from "./fluidRunner";
```

Comment:
Should we use explicit lists for any new exports we write from an `index.ts` file? I get this might be an internal tool and thus we can be more flexible, but I'd lean towards doing it here anyway.